src/rpc/mempool.cpp

```diff
@@ -217,6 +218,9 @@ static RPCHelpMan testmempoolaccept()
                         result_inner.pushKV("vsize", virtual_size);
                         UniValue fees(UniValue::VOBJ);
                         fees.pushKV("base", ValueFromAmount(fee));
+                        if (fee != tx_result.m_effective_feerate.value().GetFee(virtual_size)) {
```

Comment:
In 57b56b7ce9ebef646d3fe3e706bf994c17864f3c "[rpc] return effective-feerate in testmempoolaccept and submitpackage"

For here and in `submitpackage`, it's not clear to me why `effective-feerate` and `effective-includes` should be guarded by whether it is different from the base fees. The presence (or lack thereof) of this field indicates whether this transaction was validated as part of the package, but that is also more explicitly represented by `effective-includes`. I don't think it's harmful to provide this information for all transactions.